spec/expeditor/rich_future_spec.rb

```diff
@@ -226,14 +226,12 @@
           max_threads: 1,
           max_queue: 1,
         )
-        future1 = Expeditor::RichFuture.new(executor: executor) do
-          42
-        end
-        future2 = Expeditor::RichFuture.new(executor: executor) do
-          42
+        futures = 3.times.map do
+          future1 = Expeditor::RichFuture.new(executor: executor) do
```

Comment:
> (question) Why assign future1 variable?

Oops, it's just a mistake. Removed in https://github.com/k0kubun/expeditor/commit/a9fc25a8b1073fda51a9cd96bbd00c8f4ce05560.

> (nits) I prefer Array.new(3) than 3.times.map to explain that building an array of 3 elements.

I have no strong opinion about it. But since some other places write in this way, I want to unify the way to write this.